example/express.js

```diff
@@ -6,9 +6,7 @@ let parallel = require('run-parallel')
 let rpc = require('./rpc')
 
 function Hex256bit (value) {
-  return typeof value === 'string' &&
-    /^([0-9a-f]{2})+$/i.test(value) &&
-    value.length === 64
+  return /^[0-9a-fA-F]{64}$/.test(value)
```

Comment:
you shouldn't remove `typeof`,  see `/^[0-9a-fA-F]{3}$/.test(111)`,  yeah,  you probably can't get the JS to interpret it as a string of length 64, but, still.